### DISCUSSION TITLE
Set Minimum bounds for Individual Entities

### DISCUSSION THREAD

**Discussion by maxroxx** [kalkih/mini-graph-card#1049] [General]
Hey,
Just wanted to check whether its possible to set minimum bounds for individual entities separately otherwise temperature and humidity lines look pretty much straight.
So is it possible to set them individually or i will have to set them up as separate cards ?
<img width="484" alt="Screenshot 2024-01-05 at 12 40 25 PM" src="https://github.com/kalkih/mini-graph-card/assets/26378317/e0d46338-6c1b-49f4-bbdf-9fc38d1b6657">

**Comment by ildar170975** [comment]
No code posted.
And consider explain what "minimum bounds" are.

**Reply by maxroxx** [reply]
`type: custom:mini-graph-card
entities:
  - entity: sensor.zhimi_mb5_f0f6_pm25_density
    name: PM 2.5 Density
  - entity: sensor.zhimi_mb5_f0f6_temperature
    name: Temerature
  - entity: sensor.zhimi_mb5_f0f6_relative_humidity
    name: Humidity
    height: 10
name: Environment
hours_to_show: 8
points_per_hour: 2
show:
  fill: fade
  labels: false
  average: true
  extrema: true`


here's the code, and as per my understanding upper and lower bounds are something through which you can define the range on y axis and i believe by the default its dynamic that it sets accordingly to the readings it gets from sensor but since pm 2.5 levels range way bigger compared to temperature or humidity, these two looks pretty much like a straight line on the graph but if i set it with using upper lower bound options then it gets applied to all three entities which messes with other two readings.

**Reply by ildar170975** [reply]
1. Format the posted code since it is impossible to analyse it.
Compare with other posts with codes.
2. Entities with different UoMs & probably different ranges are definitely not supposed to be shown in the same Y-axis. Nobody does it in a real life with a paper & pencil. Please read about secondary Y-axis in Docs which may be used to display a graph with a different UoM. And the entity with the 3rd UoM is definitely not supposed to be shown on the same card.

So, what you can do is:
1. Display only 2 graphs on a card (for instance, pm25 & temperature). Place the 2nd entity on the secondary Y-axis.
2. Define lower_bound & upper_bound separately for the main & secondary Y-axis if needed.
3. The 3rd entity (humidity) may be either shown on another card - or on the same card as just a state (w/o graph).

**Comment by maxroxx** [comment]
@ildar170975 ohhh yeah silly me was only thinking it range wise, it just now hit me that those three sensors are showing readings in completely three different UoM.
Thanks for suggestions.

**Reply by ildar170975** [reply]
You are welcome.
Suggest to ask similar questions here: https://community.home-assistant.io/t/lovelace-mini-graph-card/71385/.
Could be good to share you thoughts in this forum.
